server/app.ts

```diff
@@ -16,4 +68,37 @@ const server = http.createServer((req, res) => {
 
 server.listen(3000, () => {
     console.log('Server is running on port 3000');
-});
\ No newline at end of file
+});
+
+function pad(num: number) {
+    if (num > 99) {
+        return `#${num}`
+    }
+
+    if (num > 9) {
+        return `#0${num}`
+    }
+
+    return `#00${num}`
+}
+
+function pokeAPIPokemonToAPIPokemon(pokeApiPokemon: Pokemon): APIPokemon {
+    return {
+        pokemonName: pokeApiPokemon.name,
+        pokemonNumber: pad(pokeApiPokemon.id),
+        pokemonTypes: pokeApiPokemon.types.map(({type:{name}})=> mapToAPIPokemonType(name)),
+        boxBg: COLORS[mapToAPIPokemonType(pokeApiPokemon.types[0].type.name)],
+        pokemonSvg: `https://github.com/eidan66/pokemon-api-sprites/blob/master/sprites/pokemon/other/showdown/${pokeApiPokemon.id}.gif?raw=true`
```

Comment:
Can you return also `pokemonId` please?
```suggestion
        pokemonName: pokeApiPokemon.name,
        pokemonId:pokeApiPokemon.id),
        pokemonNumber: pad(pokeApiPokemon.id),
        pokemonTypes: pokeApiPokemon.types.map(({type:{name}})=> mapToAPIPokemonType(name)),
        boxBg: COLORS[mapToAPIPokemonType(pokeApiPokemon.types[0].type.name)],
        pokemonSvg: `https://github.com/eidan66/pokemon-api-sprites/blob/master/sprites/pokemon/other/showdown/${pokeApiPokemon.id}.gif?raw=true`
```